src/main/java/edu/rpi/legup/ui/LegupUI.java

```diff
@@ -139,8 +139,9 @@ public void errorEncountered(String error) {
         JOptionPane.showMessageDialog(null, error);
     }
 
+    // TODO: implement
     public void showStatus(String status, boolean error, int timer) {
-        // TODO: implement
```

Comment:
Adding `TODO` comments inside methods is standard (not where a header comment would be placed).